samples/database.sample.yml

```diff
@@ -0,0 +1,670 @@
+# Mappings to SQLLite Database
+#
+# FST lemma             = lemma.cw
+# FST stem              = fstStem
+# homograph key         = key
+# head                  = lemma.sro
+# inflectional category = senses > sense.category
+
+- fstStem:          kinêpiko-maskotêw
+  generalWordClass: N
+  key:              kinêpiko-maskotêhk
+  lemma:
+    sro: kinêpiko-maskotêhk
+    cw:  kinêpiko-maskotêhk
+  literalMeaning: at snake plain
+  senses:
+    - category:   INM
+      definition: Mistawasis Reserve, Mistawasis First Nation, SK
+    - category:   INM
+      definition: Plains Cree community
+  specificWordClass: INM
+
+- generalWordClass: Part
+  key:              piskihci
+  lemma:
+    sro: piskihci
+    cw:  piskihci
+  senses:
+    - category:   IPC
+      definition: separate
+    - category:   IPC
+      definition: by itself
+  specificWordClass: IPC
+
+- generalWordClass: Part
+  key:              anohc kâ-kîsikâk
+  lemma:
+    sro: anohc kâ-kîsikâk
+    cw:  anohc kâ-kîsikâk
+  senses:
+    - category:   IPH
+      definition: today
+  specificWordClass: IPH
+
+- generalWordClass: Part
+  key:              kitatamihin
+  lemma:
+    sro: kitatamihin
+    cw:  kitatamihin
+  senses:
+    - category:   IPJ
+      definition: thank you, you make me smile, you please me
+  specificWordClass: IPJ
+
+- fstStem:          okakihcimosk
+  generalWordClass: N
+  key:              okakihcimosk
+  lemma:
+    sro: okakihcimosk
+    cw:  okakihcimosk
+  senses:
+    - category:   NA-1
+      definition: boaster, braggart
+  specificWordClass: NA
+
+- fstStem:          opîkopicikêw
+  generalWordClass: N
+  key:              opîkopicikêw
+  lemma:
+    sro: opîkopicikêw
+    cw:  opîkopicikêw
+  senses:
+    - category:   NA-2
+      definition: one who ploughs
+  specificWordClass: NA
+
+- fstStem:          kiskinwahamâtôtâpânâskw
+  generalWordClass: N
+  key:              kiskinwahamâtôtâpânâsk
+  lemma:
+    sro: kiskinwahamâtôtâpânâsk
+    cw:  kiskinwahamâtôtâpânâsk
+  senses:
+    - category:   NA-3
+      definition: school bus
+  specificWordClass: NA
+
+- fstStem:          oski-ây
```

Comment:
The fststems should be whatever we specify them in crk-db, and their formatting specifying according to crk conventions - the formatting would certainly vary from language to language.

In crk, stems are not always equivalent to independent word-forms - this is explicitly indicated by hyphenation. All stems are affixed a final hyphen, but in addition, stems of dependent nouns are affixed with an initial hyphen.

Reply:
Thanks for clarifying - I was going to ask you about this. I'll add the leading/trailing hyphens.